### PR TITLE
Add calendar component

### PR DESCRIPTION
**Pull request by hwangange**
https://github.com/blackgirlbytes/LunaFocus/assets/12180288/8cfaa1a5-f87d-44ed-85f9-dc4ecb87b7ba

### CODE REVIEW

**Comment by blackgirlbytes** [comment]
Cool! This works for me in the preview and locally!
One clarifying question: for now, this PR is only supposed to add random events? It doesn't handle adding specific events for specific dates yet? Is that correct?

**Review by blackgirlbytes — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fullcalendar/core": "^6.1.11",
```
Comment (blackgirlbytes) [inline package.json]: do you need this dependency? I don't see it used throughout the app. I only see fullcalendar/react and fullcalendar/daygrid being used. I don't know for sure..just curious

**Comment by hwangange** [comment]
> Cool! This works for me in the preview and locally! One clarifying question: for now, this PR is only supposed to add random events? It doesn't handle adding specific events for specific dates yet? Is that correct?

Not yet, just wanted to verify that we can properly add events without reloading the page.

File: package.json (hunk 1 above)
Comment (hwangange) [reply package.json]: Removed unused dependency

**Pull request merged**